client_test/rpc_test.go

```diff
@@ -51,28 +51,24 @@ func TestRpcClient(t *testing.T) {
 	msgB.AddPeers(peers)
 	msgI.AddPeers(peers)
 
-	defer msgA.Close()
-	defer msgB.Close()
-	defer msgI.Close()
-
-	rpcServerA := rpc.NewRpcServer(&clientA, chainId, createLogger(logDestination, "alice", "server"))
-	rpcClientA, err := rpc.NewRpcClient(rpcServerA.Url(), alice.Address(), chainId, createLogger(logDestination, "alice", "client"))
-	if err != nil {
-		t.Fatal(err)
-	}
+	defer cleanupFnA()
+	defer cleanupFnB()
+	defer cleanupFnC()
 
-	defer rpcServerA.Close()
 	defer rpcClientA.Close()
+	defer rpcClientB.Close()
```

Comment:
Are these `Close` calls needed given the cleanup functions?

Reply:
Good catch. No, they are not.

Reply:
[2f3f24c](https://github.com/statechannels/go-nitro/pull/1123/commits/2f3f24c82be900d5b06d8f570be51e07a82142fa)